resources/views/backend/posts/index.blade.php

```diff
@@ -50,10 +51,10 @@
                   </tr>
                 </thead>
                 <tbody>
-                  @foreach ($posts as $post)
+                  @foreach ($posts as $index => $post)
                     <tr>
-                      <td class="text-center">{{ $post->id }}</td>
-                      <td><a href="{{ route('posts.show', $post->id) }}">{!! Str::words($post->content, config('define.posts.size_short_content'),config('define.posts.three_dots')) !!}</a></td>
+                      <td class="text-center">{{ $index + $posts->firstItem() }}</td>
+                      <td><a href="{{ route('posts.show', [$post->id, 'page' => $_GET['page'] ?? 1]) }}">{!! Str::words($post->content, config('define.posts.size_short_content'),config('define.posts.three_dots')) !!}</a></td>
```

Comment:
Không được sử dụng `$_GET` vs `$_POST` thuần của PHP nha, vì nó ko an toàn và bảo mật kém.
Dùng `request('page')` của Laravel á.